File: week 1/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //list of variables used
    int h;
    int i;
    int j;
    int k;
    //request height from user until between 1-8
    do
    {
        h = get_int("Pyramid height: ");
    }
    while (h < 1 || h > 8);
    //loop to build pyramid based on height (h)
    for (i = 1; i <= h; i++)
    {
        for (k = h; k > i; k--)
        {
            //spaces on pyramids left
            printf(" ");
        }
        for (j = 1; j < i; j++)
        {
            //hashes from the pyramids right
            printf("#");
        }
        //last hash and line break
        printf("#\n");
    }
}

File: week 6/sentimental-mario-less/mario.py
from cs50 import get_int 

# Initialise height value
height = 0 

# Ensure height input from user is between 1 and 8
while height < 1 or height > 8:
    height = get_int("how tall is the pyramid? ")

# Print out spaces and hash's for pyramid
for i in range(1, height + 1):
    print(" " * (height - i) + ("#" * i))